content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -8,66 +8,80 @@ card:
 ---
 
 <!-- overview -->
-ConfigMaps allow you to decouple configuration artifacts from image content to keep containerized applications portable. This page provides a series of usage examples demonstrating how to create ConfigMaps and configure Pods using data stored in ConfigMaps.
-
+{{< glossary_tooltip text="ConfigMaps" term_id="configmap" >}} allow you to decouple
+configuration data from container image artifacts to keep containerized applications
+portable. This page provides a series of usage examples demonstrating how to create
+`ConfigMap`s and configure `Pod`s using data stored in `ConfigMap`s.
```

Comment:
@sftim, In reading the page again, the mix of fonts does not look quite right:
`ConfigMap`s and `Pod`s.
What do you think?

Reply:
I'm minded to go with this for now, but use the poor rendering that comes from following the current style guide to support a case for reverting the style guide so that we _don't_ write API object kinds in code style.

The thing you configure to use a ConfigMap _is_ a Pod, the API object. That's true no matter what font you write it in!

Reply:
OK, though this could introduce more updates.
You could leave the existing text (ConfigMaps and configure Pods) and instead create an issue to modify the style guide explaining the problems.

Reply:
?